Milestone 1

li-kai opened the pull request:
Fixes #67, #64, #63, #62

li-kai commented:
Deploy preview ready!

Built with commit af096319bdbcc1a86a3d3b9282087d3f79f083cd

https://deploy-preview-74--nushackers.netlify.com

donjar commented:
Great job @li-kai!! 👏 

Was wondering about the "Friday Hacks" section in the front page though. I find it a bit weird clicking "Show more" shows from 0 to 2. Maybe change it to "show" instead, and maybe add a link to show all the Friday Hacks?

Also, not sure how I feel about https://deploy-preview-74--nushackers.netlify.com/about/. Since the sidebar is at the left, I am not very sure, if I am a newcomer where should I read the page first?

li-kai commented:
@donjar It was on the left for the original, which I added before redesign. Switched to the other side.

Now the the list should show at least 2, and only a "show more" if there is more... The full list is at /fh/, but I don't see any point of the full list when we have no taxonomies yet. Milestone 2 I guess.
